tests/merlin_hashtable_test.cc.cu

```diff
@@ -1944,6 +1944,367 @@ void test_evict_strategy_customized_correct_rate(size_t max_hbm_for_vectors,
 
   CudaCheckError();
 }
+
+void test_insert_or_assign_multi_threads(
+      size_t max_hbm_for_vectors, 
+      bool use_constant_memory, 
+      const uint64_t init_size_in_mb, 
+      const uint64_t max_size_in_mb,
+      const uint64_t THREAD_N,
+      const uint64_t BATCH_0_SIZE,
+      const uint64_t BATCH_1_SIZE,
+      bool capacity_silent = true
+    ) {
+  const uint64_t INIT_CAPACITY = nv::merlin::MB(init_size_in_mb);
```

Comment:
As I mentioned in the meeting, the unit of capacity is key-value pairs, not `bytes`.